src/sentry/integrations/github/repository.py

```diff
@@ -65,25 +36,25 @@ def validate_config(self, organization, config):
         return config
         ```
         """
-        if config.get('name') and config.get('installation'):
+        if config.get('identifier') and config.get('installation'):
```

Comment:
Should this throw an error or something if `'identifier'` and `'installation'` aren't present? Why is there an if statement? Will it work otherwise? Also noting it was like this before.

Reply:
probs should do similar to what you have for VSTS - I can change